### ISSUE TITLE
Update installation instructions to current practice

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/use_bluesky#119]
Time to update the installer script. What it says now is out-of-date: https://github.com/BCDA-APS/use_bluesky/blob/288aa9032c26d548ef322a0f7cfce446ff5231dc/install/install_startup.sh#L4-L7

Now using a directory layout like this one:

```text
~/
  bin/
    blueskyStarter.sh  --> ~/bluesky/blueskyStarter.sh
  bluesky/
    blueskyStarter.sh
    console/
      __start_bluesky_instrument__.py
    instrument/
  .ipython-bluesky/
    profile_bluesky/
      startup/
        __start_bluesky_instrument__.py --> ~/bluesky/console/__start_bluesky_instrument__.py
```

and `__start_bluesky_instrument__.py` contains:

```py
"""
start bluesky in IPython session
"""

# start a Bluesky data collection console session
import pathlib, sys
sys.path.append(str(pathlib.Path.home()))

# terse error dumps
get_ipython().run_line_magic('xmode', 'Minimal')

from instrument.collection import *
```

**Comment by prjemian** [comment]
@mdwyman Thanks for the incentive.

**Comment by prjemian** [comment]
Deposit the `__start_bluesky_instrument__.py` file into https://github.com/BCDA-APS/bluesky_training/tree/main/bluesky/resources

It should not go into the parent directory since **that** directory _must remain compatible with the bluesky-queueserver_ and have only one `.py` file.

**Comment by prjemian** [comment]
Also, update the `instrument_package_guide.md` file to include use and maintenance of `instrument/iconfig.yml`.

**Comment by mdwyman** [comment]
@prjemian I worked through the new install instructions but early on had to use micromamba instead of conda on the install bluesky environment step (https://github.com/BCDA-APS/use_bluesky/tree/119-update-install-advice/install#install-bluesky-environment). If conda was going to "solve" the environment, it would have taken >30 min (I stopped after this much elapsed).  I found the micromamba instructions in the yaml file comments and it seemed to finish in <5 min.  Are we moving away from conda? Should we make this explicit?

I was doing this from the APS network on beamline machines and a BC machine -- none of the conda environments were solved within 30 min.  I tried several of the environments (2021_2, 2022_1, 2022_2, 2022_3) to same effect

**Comment by prjemian** [comment]
Exactly. I was testing this out in August, coming to the same conclusions
as you. So I installed micromamba in the base environments on APSshare. The
downside is that it uses different directories than conda for local
environments.

So, yes, use micromamba instead of conda now.
